sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/healthcare/FhirIO.java

```diff
@@ -17,6 +17,8 @@
  */
 package org.apache.beam.sdk.io.gcp.healthcare;
 
+import static org.apache.beam.sdk.io.gcp.healthcare.FhirIO.ExecuteBundles.FAILED_BUNDLES;
```

Comment:
This looks like we're importing from the current file which I feel like shouldn't be necessary. Can we define these in an outer class so we have broader access to them, or make them not private?

Reply:
Done